pom.xml

```diff
@@ -32,7 +32,7 @@
     <repository.name>hygieia-jenkins-cucumber-collector</repository.name>
     <apache.rat.plugin.version>0.13</apache.rat.plugin.version>
     <bc.version>3.0.1</bc.version>
-    <com.capitalone.dashboard.core.version>3.1.5</com.capitalone.dashboard.core.version>
+    <com.capitalone.dashboard.core.version>3.10.0</com.capitalone.dashboard.core.version>
     <commons.io.version>2.4</commons.io.version>
     <commons.lang.version>3.8.1</commons.lang.version>
     <coveralls.maven.plugin.version>4.3.0</coveralls.maven.plugin.version>
```

Comment:
since the parent is changed, the deploy to maven central would fail. Can you copy the sections `<build>, <reporting> and <profiles>` from [hygieia-build-jenkins-collector]( https://github.com/Hygieia/hygieia-build-jenkins-collector/blob/master/pom.xml) so that the release can be done.

Reply:
Of course, the requested changes have been added. Let me know if there is anything else that needs to be changed.